package.json

```diff
@@ -21,7 +21,7 @@
     "dev": "next -- -p 3001",
     "build": "next build",
     "start": "next start",
-    "test": "react-scripts test",
+    "test": "react-scripts test --setupFiles ./src/setupTests.js",
```

Comment:
- [ ] Our project should be setup to do this step automatically. You can remove `--setupFiles ./src/setupTests.js`

Reply:
Hi @rgavinc. I initially didn't intend to place it there (I had a `jest.config.json` file at the project root previously and running the tests went well). However, for this particular PR I was experiencing some errors in the pre-commit tests so I experimented with some solutions and this was the one that worked (meaning, no errors for both regular running of `npm run test` and in the pre-commit tests). Kindly help me on how to solve this

Reply:
@rvvergara  Try pulling down the most recent changes and then see if you are still having difficulties running  `npm run test` without `--setupFiles ./src/setupTests.js`. It should be working without that now but if it isn't please let me know. It was working when I tried it.

Reply:
Ok I pulled the changes (rebased based on changes in `master`). So then the previous `test` script I wrote on `package.json` was reverted back to the previous one and I also brought back `jest.config.json`. But then I encounter this error:

![image](https://user-images.githubusercontent.com/6015315/68631589-515a7400-0526-11ea-8872-aa607d3a3108.png)

![image](https://user-images.githubusercontent.com/6015315/68631796-11e05780-0527-11ea-9756-263f0c0d43c9.png)

Reply:
Ok, so what I did, just so I can commit is I included the `--no-verify` flag when I committed. Though I am not comfortable with the fact that there's still an error when not using that flag.